inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndex.java

```diff
@@ -621,12 +625,17 @@ public void createPhysicalIndex()
             FileUtils.forceMkdir(indexDir);
 
             // Open the index
+            log.info("Opening index for project " + project.getName());
```

Comment:
I would again use string formatting when logging.

Reply:
With SLF4J, concatenation should definitely be avoided. It can have similar undesired effects as code injection, e.g. if the concatenated string contains `{}` (not the case here, but still).

Reply:
OK.